aries_vcx/src/protocols/issuance_v2/holder/states/credential_received.rs

```diff
@@ -0,0 +1,29 @@
+use messages::msg_fields::protocols::cred_issuance::v2::issue_credential::IssueCredentialV2;
+
+use crate::protocols::issuance_v2::formats::holder::HolderCredentialIssuanceFormat;
+
+pub struct CredentialReceived<T: HolderCredentialIssuanceFormat> {
+    #[allow(dead_code)] // `credential` may become used in future
+    pub(crate) credential: IssueCredentialV2,
+    pub(crate) stored_credential_metadata: T::StoredCredentialMetadata,
```

Comment:
Perhaps should be private. We are actually not using `new` constructor now, as it's possible to construct the `struct` manually - which is what we do instead.